portfolio/src/main/webapp/js/background.js

```diff
@@ -16,9 +30,10 @@ const INITIAL_BALL_VELOCITY_MAGNITUDE = 1;
 function setup() {
   const backgroundContainer = document.getElementById("background-container");
 
+  const tankHeightToWidth = TANK_HEIGHT / TANK_WIDTH;
```

Comment:
Since this is used here and in `windowResized` below, would it make sense to have another constant above `TANK_HEIGHT_TO_WIDTH`?

Reply:
That does make sense. Will change that.